Finalize Integrations (#152)
nautobot/nautobot-plugin-ssot: README.md

* The outcome of executing of a data synchronization Job is automatically saved to Nautobot's database for later review.
* Detailed logging output generated by DiffSync is automatically captured and saved to the database as well.

### Integrations

> This plugin supports the following integrations:

Reviewer: ```suggestion
This Nautobot application framework includes the following integrations:
```

* The outcome of executing of a data synchronization Job is automatically saved to Nautobot's database for later review.
* Detailed logging output generated by DiffSync is automatically captured and saved to the database as well.

### Integrations

> This plugin supports following integrations:

Reviewer 2: ```suggestion
This plugin supports the following integrations:
```

- Cisco ACI
- Arista CloudVision
- Infoblox
- IPFabric
- ServiceNow

> Read more about integrations [here](https://docs.nautobot.com/projects/ssot/en/latest/user/integrations).
> 
> To install integrations follow [this guide](https://docs.nautobot.com/projects/ssot/en/latest/admin/install/#integrations-configuration).

Reviewer 2: ```suggestion
Read more about integrations [here](https://docs.nautobot.com/projects/ssot/en/latest/user/integrations). To enable and configure integrations follow the instructions from [the install guide](https://docs.nautobot.com/projects/ssot/en/latest/admin/install/#integrations-configuration).
```

## Try it out!

This Nautobot app is installed in the Nautobot Community Sandbox found over at [demo.nautobot.com](https://demo.nautobot.com/)!

> For a full list of all the available always-on sandbox environments, head over to the main page on [networktocode.com](https://www.networktocode.com/nautobot/sandbox-environments/).

## Documentation

Full documentation for this app can be found over on the [Nautobot Docs](https://docs.nautobot.com) website:

- [User Guide](https://docs.nautobot.com/projects/ssot/en/latest/user/app_overview/) - Overview, Using the App, Getting Started.
- [Administrator Guide](https://docs.nautobot.com/projects/ssot/en/latest/admin/install/) - How to Install, Configure, Upgrade, or Uninstall the App.
- [Developer Guide](https://docs.nautobot.com/projects/ssot/en/latest/dev/contributing/) - Extending the App, Developing Jobs, Code Reference, Contribution Guide.
- [Release Notes / Changelog](https://docs.nautobot.com/projects/ssot/en/latest/admin/release_notes/).

### Contributing to the Documentation

> You can find all the Markdown source for the app documentation under the [`docs`](https://github.com/nautobot/nautobot-plugin-ssot//tree/develop/docs) folder in this repository. For simple edits, a Markdown capable editor is sufficient: clone the repository and edit away.

Author: Fixed

## Questions

For any questions or comments, please check the [FAQ](https://docs.nautobot.com/projects/ssot/en/latest/user/faq/) first. Feel free to also swing by the [Network to Code Slack](https://networktocode.slack.com/) (channel `#nautobot`), sign up [here](http://slack.networktocode.com/) if you don't have an account.

## Acknowledgements

> This project includes code originally written in separate plugins, which have been merged into this project:

Reviewer: ```suggestion
This project includes code originally written in separate Nautobot apps, which have been merged into this project:
```

## Questions

For any questions or comments, please check the [FAQ](https://docs.nautobot.com/projects/ssot/en/latest/user/faq/) first. Feel free to also swing by the [Network to Code Slack](https://networktocode.slack.com/) (channel `#nautobot`), sign up [here](http://slack.networktocode.com/) if you don't have an account.

## Acknowledgements

This project includes code originally written in separate plugins, which have been merged into this project:

- [nautobot-plugin-ssot-aci](https://github.com/nautobot/nautobot-plugin-ssot-aci):
    Thanks
    [@chadell](https://github.com/chadell),
    [@dnewood](https://github.com/dnewood),
    [@progala](https://github.com/progala),

>     [@ubajze](https://github.com/ubajze),

Reviewer 2: ```suggestion
    [@ubajze](https://github.com/ubajze)
```
